build/x86_64_groovy/Dockerfile

```diff
@@ -24,5 +24,8 @@ RUN apt-get -y update \
         make \
         pkg-config \
         python-dev-is-python2 \
+ && echo 'deb http://archive.ubuntu.com/ubuntu/ focal main' >/etc/apt/sources.list.d/focal.list \
```

Comment:
Let's add a comment before this, something like:
`# Work around https://gitlab.archlinux.org/archlinux/archlinux-docker/-/issues/32.`
?